### PR TITLE
Fix mypy type annotations

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

File: xiblint/rules/accessibility_format.py [hunk 1]
```diff
@@ -7,12 +7,15 @@
     view_accessibility_identifier,
 )
 
+if False:  # if TYPE_CHECKING:
```
Comment (kastiglione) [inline xiblint/rules/accessibility_format.py]: How does this work? Should there be a follow up to remove `if False`? Or is mypy reading the comment? I don't see anything about commenting it out here: https://mypy.readthedocs.io/en/latest/common_issues.html

Comment (asottile) [reply xiblint/rules/accessibility_format.py]: It is equivalent to https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

however, `typing.TYPE_CHECKING` is not available in python3.5 and I wasn't sure what versions of python `xiblint` supports

`mypy` (and other static type checkers) follow that import, but at runtime it doesn't occur.

I wasn't sure if the import was cyclical or I wouldn't have guarded it by `if False:`, if it's not a cycle the `if False` can be removed entirely

Comment (kastiglione) [reply xiblint/rules/accessibility_format.py]: Ah so, mypy evaluates all `import`s in a file unconditionally?

Comment (asottile) [reply xiblint/rules/accessibility_format.py]: not all, but those that equate to `if False:` or `if typing.TYPE_CHECKING:` or `if sys.version (comparison) (compatible version):` and  a few other cases like that

Comment (kastiglione) [reply xiblint/rules/accessibility_format.py]: thanks for mypy education

**Comment by ikonst** [comment]
Thanks.

I think the `if TYPE_CHECKING:`s are due to https://github.com/PyCQA/pyflakes/issues/247 so we can try and remove them.

**Pull request merged**